python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1078,6 +1078,13 @@ def test_dataframe_query():
     tm.assert_equal(actual, expected)
 
 
+def test_private_method_result_wrapped():
+    xoffset = xpd.offsets.Day()
+    dt = datetime.datetime(2020, 1, 1)
+    result = xoffset._apply(dt)
+    assert type(result) is xpd.Timestamp
```

Comment:
Is this semantically equivalent for this case?
```suggestion
    assert isinstance(result, xpd.Timestamp)
```

Reply:
Yes I believe so (I am not sure why I didnt use this in the first place)